index.html

```diff
@@ -4975,6 +4975,634 @@ <h2 id="AudioWorkerParamDescriptor">
+              constructor. Entries in this dictionary whose names correspond to
+              <a><code>AudioParam</code></a>s in the class definition of an
+              <a><code>AudioWorkletProcessor</code></a> are used to initialize
+              the parameter values upon the creation of a node.
+            </p>
+            <dl title="dictionary AudioWorkletNodeOptions" class="idl">
+              <dt>
+                unsigned long numberOfInputs
+              </dt>
+              <dd>
+                Same to <a>numberOfInputs</a> in <a><code>AudioNode</code></a>.
+              </dd>
+              <dt>
+                unsigned long numberOfOutputs
+              </dt>
+              <dd>
+                Same to <a>numberOfOutputs</a> in
+                <a><code>AudioNode</code></a>.
+              </dd>
+              <dt>
+                unsigned long channelCount
+              </dt>
+              <dd>
+                Same to <a>channelCount</a> in <a><code>AudioNode</code></a>.
+              </dd>
+              <dt>
+                ChannelCountMode channelCountMode
+              </dt>
+              <dd>
+                Value used to initialize the
+                <a><code>channelCountMode</code></a> attribute of an
+                <a><code>AudioWorkletNode</code></a>
+              </dd>
+              <dt>
+                ChannelInterpretation channelInterpretation
+              </dt>
+              <dd>
+                Same to <a>channelInterpretation</a> in <a>AudioNode</a>.
+              </dd>
+            </dl>
+          </section>
+        </section>
+        <section>
+          <h2 id="AudioWorkletGlobalScope">
+            The <dfn>AudioWorkletGlobalScope</dfn> interface
+          </h2>
+          <p>
+            This interface is derived from <code>WorkletGlobalScope</code> and
+            represents a special execution context in which an audio processing
+            script runs. It is designed to enable the generation, processing,
+            and analysis of audio data directly using JavaScript in the audio
+            <a>rendering thread</a>. The user-supplied script code is evaluated
+            in this scope to define an
+            <a><code>AudioWorkletProcessor</code></a>.
+          </p>
+          <dl title="interface AudioWorkletGlobalScope : WorkletGlobalScope"
+          class="idl">
+            <dt>
+              void registerAudioWorkletProcessor (DOMString name, Function
+              classConstructor)
+            </dt>
+            <dd>
+              <p>
+                Registers a definition of a class that is derived from
+                <a>AudioWorkletProcessor</a>.
+              </p>
+              <dl class="parameters">
+                <dt>
+                  DOMString name
+                </dt>
+                <dd>
+                  A string key that represents a class definition to be
+                  registered.
```

Comment:
For clarity mention that this key is used to resolve an AWP during construction of an AWN.